### ISSUE TITLE
Fix Cloud cost credentials leftovers

### ISSUE THREAD

**Issue by eranelbaz** [env0/terraform-provider-env0#335]
Since we want to unblock https://github.com/env0/terraform-provider-env0/pull/307 we decided to make this refactor in another issue,
Let's try to refactor tests and DRY it out

**Comment by yaronya** [comment]
@eranelbaz @samuel-br also please add missing examples + integration tests for these resources & data resources

**Comment by yaronya** [comment]
Also, what about import? 
@eranelbaz

**Comment by eranelbaz** [comment]
> Also, what about import? @eranelbaz

We need to fix / add `import` for all credentials, we got stuck on - https://github.com/env0/terraform-provider-env0/issues/99 

IDK if important

**Comment by TomerHeber** [comment]
@eranelbaz - I think import was added in #99 for AWS, Azure, and GCP.
If it covers all use-cases, can this issue be closed?

**Comment by eranelbaz** [comment]
The idea here is to try and DRY the test cases since they are the same tests 3 times,
one for each cloud provider

**Comment by TomerHeber** [comment]
might be related to https://github.com/env0/terraform-provider-env0/issues/394

**Comment by yaronya** [comment]
Yes @TomerHeber 
I'm closing it as it's fixed in #394
